include/ignition/physics/Joint.hh

```diff
@@ -382,6 +383,57 @@ namespace ignition
         public: virtual void DetachJoint(const Identity &_jointID) = 0;
       };
     };
+
+    class IGNITION_PHYSICS_VISIBLE GetJointTransmittedWrench
+        : public virtual FeatureWithRequirements<JointFrameSemantics>
+    {
+      public: template <typename PolicyT, typename FeaturesT>
+      class Joint
+          : public virtual JointFrameSemantics::Joint<PolicyT, FeaturesT>
+      {
+        public: using Wrench = typename FromPolicy<
+                    PolicyT>::template Use<Wrench>;
+
+        /// \brief Get the transmitted wrench at the Joint frame.
+        ///
+        /// The transmitted wrench is the force and torque
+        /// applied by the parent link on the child link, transmitted through
+        /// the joint. It is the sum of constraint forces from the joint,
+        /// applied joint force (set by the user using the Joint::SetForce API)
+        /// as well as forces due to joint friction, damping, and spring
+        /// stiffness.
+        public: Wrench GetTransmittedWrench() const;
+
+        /// \brief Get the transmitted wrench of this joint at the specified
+        /// reference frame and expressed at the specified coordinate frame.
+        ///
+        /// The transmitted wrench is the force and torque applied by the parent
+        /// link on the child link, transmitted through the joint. It is the sum
+        /// of constraint forces from the joint, applied joint force (set by the
+        /// user using the Joint::SetForce API) as well as forces due to joint
+        /// friction, damping, and spring stiffness.
+        /// \param[in] _referenceFrame Reference frame in which the wrench is
+        /// expressed. Note that the wrench is updated such that linear force
+        /// component is applied at the origin of this reference frame.
+        /// \param[in] _inCoordinatesOf Coordinate frame in which the wrench is
+        /// expressed. Unlike _referenceFrame, the coordinate frame is only used
+        /// to apply a rotation to the individual vectors in the wrench. It does
+        /// not move the point where the force is applied.
+        public: Wrench GetTransmittedWrench(
+                    const FrameID &_referenceFrame,
```

Comment:
consider naming variable `_relativeTo` to match the parameter name in `detail::Resolve` from FrameSemantics.hh

Reply:
Done in https://github.com/ignitionrobotics/ign-physics/pull/283/commits/8709364a018a32f2c3ac863a07c31bd465da008b